exotic/exotic.py

```diff
@@ -217,6 +217,10 @@ def julian_date(hdr, time_unit, exp):
 
     return julian_time + offset
 
+def get_exp_time(hdr):
+    exp_list = [ "EXPTIME", "EXPOSURE", "EXP" ]
+    exp_time = next((exptime for exptime in exp_list if exptime in hdr), None)
+    return hdr[exp_time] if exp_time is not None else 0.0
```

Comment:
Looks good to me. I'm not too sure if we should leave the default case to 0 seconds, but I can't think of a better alternative currently. Nice find!